docs/csharp/programming-guide/classes-and-structs/private-constructors.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Private Constructors - C# Programming Guide"
+description: A private constructor is a special instance constructor in C#, generally used in classes that contain static members only.
```

Comment:
There are other good uses for private constructors. (factory methods, etc). Let's leave it open in the description.

```suggestion
description: A private constructor is a special instance constructor in C# used to restrict how an object can be created. They may be used with factory methods or other construction idioms.
```